blocks/quote/quote.js

```diff
@@ -0,0 +1,11 @@
+export default function decorate(block) {
+  const quote = block.firstElementChild.firstElementChild;
```

Comment:
I'd prefer using `querySelector` instead of this first child of the first child combination. the former is easier to maintain.